### ISSUE TITLE
Test results of official trained model(download_PieAPPv0.1_PT_weights.sh) not same with paper

### ISSUE THREAD

**Issue by JiadiMo** [prashnani/PerceptualImageError#13]
Hi, I have tested the trained model(download_PieAPPv0.1_PT_weights.sh) that you released, but the performance is lower than the result in your paper by about 2% in both PLCC, SRCC, and KRCC. Also, the reimplement is also lower than the result in your paper by about 2%. Can I ask for your testing and training details?

Thanks a lot.

**Comment by Luciennnnnnn** [comment]
@JiadiMo Hi, I also wondering how test their models, can you share your experiences?

**Comment by ch-andrei** [comment]
@JiadiMo What SROCC did you achieve? 

FYI, on my end, after a quick test with the provided pretrained weights and similar (although modified) setup
SROCC=0.79185
KROCC=0.60356
PLCC=0.78609
RMSE=0.09523
on PieAPP test set.
